modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java

```diff
@@ -1032,6 +1057,14 @@ private DDMFormInstanceVersion _getLatestApprovedDDMFormInstanceVersion()
 		return _latestDDMFormInstanceVersion;
 	}
 
+	private String _getObjectAcceptedFileExtensions(long objectFieldId) {
```

Comment:
@nicolas-sss could you please rename `_getObjectAcceptedFileExtensions` to `_getObjectFieldAcceptedFileExtensions`

Reply:
Done!